Source/RapyutaSimulationPlugins/Public/Robots/RRBaseRobot.h

```diff
@@ -182,9 +167,31 @@ class RAPYUTASIMULATIONPLUGINS_API ARRBaseRobot : public ARRBaseActor
     // UFUNCTION(BlueprintCallable)
     virtual void SetJointState(const TMap<FString, TArray<float>>& InJointState, const ERRJointControlType InJointControlType);
 
+    UFUNCTION(BlueprintCallable)
+    bool IsPossessed()
+    {
+        return bIsPossessed;
+    }
+
 protected:
     /**
      * @brief Instantiate default child components
      */
     virtual void PostInitializeComponents() override;
+
+    //! is this robot is possed or not. Defaults method's do not works with multiplayer.

```

Comment:
`Default methods` seems to be a bit vague, should it be `GetController()`?
So if `APawn::Controller` does get replicated, why didn't it show up in other clients tho? 🤔

Reply:
I'm not sure but I tried methods in the comment but do not work.
Controller replication seems not simple?
https://github.com/EpicGames/UnrealEngine/blob/d9d435c9c280b99a6c679b517adedd3f4b02cfd7/Engine/Source/Runtime/Engine/Private/Pawn.cpp#L559